bazarr/radarr/sync/movies.py

```diff
@@ -45,6 +52,16 @@ def update_movie(updated_movie, send_event):
             event_stream(type='movie', action='update', payload=updated_movie['radarrId'])
 
 
+def get_movie_monitored_status(movie_id):
+    existing_movie_monitored = database.execute(
+        select(TableMovies.monitored)
+        .where(TableMovies.tmdbId == movie_id))\
+        .first()
+    if existing_movie_monitored == None:
```

Comment:
`if not existing_movie_monitored:`

Reply:
I’m not sure that is right.
I’m doing a bit of reading and they suggest:

` if existing_movie_monitored is None:`

Reply:
In fact, the sqlalchemy may return a single row or None. My code will check if it returned something in `[None, False, 0]`. `is None` is more precise but in our specific case it wont make a difference.

Reply:
If that’s true, that would change my logic in a bad way.
I want to return True only if the db record is missing.
If the db returns False, I want to return False, not True.
I will change it to use “is None”, though.